templates/csi-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and (eq (.Values.csi.enabled | toString) "true" ) (eq (.Values.global.enabled | toString) "true") (semverCompare "> 0.0.7" .Values.csi.image.tag) }}
```

Comment:
I don't think we're guaranteed the tag wasn't modified by the user, so this check might not be enough. I don't think there's much we can do about that, though, so let's just make sure to document these requirements on the website.